spec/docx/smoke/api_uni_color_spec.rb

```diff
@@ -6,4 +6,15 @@
     docx = builder.build_and_parse('js/docx/smoke/api_uni_color/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = presetColor')
   end
+
+  it 'ApiUniColor | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_uni_color/to_json.js')
+    json = JSON.parse(docx.elements[1].nonempty_runs.first.text)
+    rgba = json['graphic']['spPr']['fill']['fill']['gsLst'][0]['color']['color']['rgba']
+    uni = json['graphic']['spPr']['fill']['fill']['gsLst'][0]['color']['type']
+    expect(uni).to eq('uniColor')
```

Comment:
Does this type change if you use RGBColor? If not, this check is trivial

Reply:
The type does not change to 'RGBColor' even if we use RGBColor directly when creating the document. This is because RGBColor is a subclass of ApiUniColor, and thus it is serialized as 'uniColor' in JSON, which is what we are verifying with this test.
The purpose of this test is to ensure that the serialization process consistently outputs the type 'uniColor' for color objects, regardless of whether the original color type in the document is RGBColor or any other subclass of ApiUniColor. Even though it might seem trivial, it is an important aspect to check to ensure the correct functioning of the serialization process.